### ISSUE TITLE
How to use custom strings?

### ISSUE THREAD

**Issue by meierrom** [qwc-services/qwc-legend-service#15]
Hi folks,

To allow returning custom legend graphics instead of the ones generated by the WMS server is a nice addition.

How about customizing the legend strings? What would be the best approach to do this?

Kind regards,
Roman

**Comment by meierrom** [comment]
Hi folks,

It may be that the question is not very clear. So let me try again. ;)

I used to work a lot with UMN Mapserver and it had a nice feature called "HTML Legends". It allowed to create customized legends incl. replacing labels.

I was wondering if something similar could be achieved using qwc-legend-service or maybe some workaround allowing to automatically create customized legend images that will then be used by qwc-legend-service?

Any ideas?

Kind regards, 
Roman

**Comment by manisandro** [comment]
The legend service will currently pick up a custom image if one is specified or forward the request to  the qgis server. It does not have any other capabilities currently.

**Comment by tpo** [comment]
Please correct me here @manisandro if I'm mistaken: pull requests for that feature and/or sponsoring the implementation of the feature is of course welcome!
